src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, createReport(readCsvFile(fromFileName)));
+    }
 
+    public void writeToFile(String file, String data) {
+        File uptputFile = new File(file);
+        FileWriter fileWriter;
+        try {
+            Files.write(uptputFile.toPath(), data.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
     }
+
+    public String createReport(String data) {
+        StringBuilder report = new StringBuilder();
+        int supply = 0;
+        int buy = 0;
+        String[] record = data.split(System.lineSeparator());
+        for (int i = 0; i < record.length; i++) {
+            if (record[i].split(",")[0].equalsIgnoreCase("buy")) {
```

Comment:
```suggestion
        for (int i = 0; i < record.length; i++) {
            String recordSplit = ecord[i].split(",");
            if (recordSplit[0].equalsIgnoreCase("buy")) {
```
split once , dont repeat unnecessary operations

Reply:
Than I will have to use an String array again not just String.
Done.